mandatory/3-function-output.js

```diff
@@ -11,6 +11,8 @@ function s(w1, w2) {
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+// return firstWord + " " + secondWord + " " +  thirdWord;
+return firstWord.concat(" ",secondWord," ", thirdWord);
```

Comment:
This is definitely correct :) But I would prefer using the + sign as it's easier to read

Reply:
The exercise is asking to use the concatenate method, unless I don't understand. Thank you